packages/plugin-scripts/bin/generate-manifest.js

```diff
@@ -1,21 +1,25 @@
+#!/usr/bin/env node
+'use strict';
+
+const path = require('path');
+const packageJson = require(path.resolve(process.cwd(), './package.json'));
+
 (async () => {
     const fs = require('fs');
 
     //1. Read from the "widgets" folder
-    const folderName = 'src/widgets';
+    const folderName = path.resolve(process.cwd(), 'src/widgets');
     const getWidgetProperties = async (widgetsList) => {
         const temp = [];
         console.log('');
         for await (const widget of widgetsList) {
             try {
-                const widget_folder_path = folderName + '/' + widget;
-                // eslint-disable-next-line node/no-unsupported-features/es-syntax
-                let WidgetProperties = await import('../' + widget_folder_path + '/WidgetProperties.json', {
-                    assert: { type: 'json' },
-                });
-                WidgetProperties.default.src = widget_folder_path;
-                temp.push(WidgetProperties.default);
-                console.info(WidgetProperties.default.name + ' widget -- READ');
+                const widget_folder_path = path.join(folderName, widget);
+                const widgetProps = require(`${widget_folder_path}/WidgetProperties.json`);
+                let WidgetProperties = JSON.parse(JSON.stringify(widgetProps));
```

Comment:
The previous code (brought in from the widgets repo) did an assertion to check that it was valid json. This serves the same purpose while also passing that eslint check that we have on this repo. If it is not valid it'll get caught in the catch.

Reply:
This won't validate the file contains valid JSON.  JSON is a string representation of some javascript value.  If what was in WidgetProperties.json wasn't valid JSON, it probably wouldn't even get here, and even if it did, the JSON has already been parsed.  At this point `widgetProps` is already referencing a valid javascript value, i.e. it's no longer JSON, and stringifying it to JSON and parsing it back will just yield the same javascript value.  This is essentially a no-op.  I think the only reason it could fail is if `widgetProps` contained circular references, in which case `stringify` might fail.

Reply:
Thanks, that answers this question I asked yesterday https://github.com/Evoke-Platform/evoke-sdk/pull/41#issuecomment-1505662676